From: SeongJae Park <sj@kernel.org>
Date: Fri, 14 Feb 2025 16:21:15 -0800
Subject: [PATCH] Docs/mm/damon/design: fix typo on DAMOS filters usage doc
 link

The link from DAMOS filters design doc to usage doc has a typo calling
filters as watermarks.  Fix it.

Fixes: d31f5626a0e1 ("Docs/mm/damon/design: add links to sections of DAMON sysfs interface usage doc")
Signed-off-by: SeongJae Park <sj@kernel.org>
---
 Documentation/mm/damon/design.rst | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Documentation/mm/damon/design.rst b/Documentation/mm/damon/design.rst
index cd3e68f8b8f4..2da0390f7012 100644
--- a/Documentation/mm/damon/design.rst
+++ b/Documentation/mm/damon/design.rst
@@ -658,7 +658,7 @@ Below ``type`` of filters are currently supported.
     - Applied to pages that belonging to a given DAMON monitoring target.
     - Handled by the core logic.
 
-To know how user-space can set the watermarks via :ref:`DAMON sysfs interface
+To know how user-space can set the filters via :ref:`DAMON sysfs interface
 <sysfs_interface>`, refer to :ref:`filters <sysfs_filters>` part of the
 documentation.
 
-- 
2.39.5

